modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -386,7 +393,6 @@ export default {
     },
     clearSelected() {
       this.checked = [];
-      this.editing = undefined;
```

Comment:
Because if `clear selection` is visible it means that `editing` is already udefined?

Reply:
the watcher on `checked` is responsible to call `this.updateEditing`